src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE DATABASE db_example;
+USE db_example;
+CREATE TABLE books (id BIGINT NOT NULL AUTO_INCREMENT,
+                    title VARCHAR(255),
+                    price DECIMAL,
+                    PRIMARY KEY (id));

```

Comment:
Primary key by default NOT NULL.
```suggestion
CREATE TABLE books (id BIGINT PRIMARY KEY AUTO_INCREMENT,
                    title VARCHAR(255),
                    price DECIMAL);

```